docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-register-solver-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-register-solver-proc']
+= Register the Solver using the REST API 
+
+Once the Solver has been created, you can register it using the REST API. By registering the Solver, you can then submit planning problems in the request body to find the optimal solution.
```

Comment:
We need to change the terminology here. Short story: "Once the solver configuration is created and the project is deployed to the decision server, you can create a solver and submit planning problems ..."

### Long story

The project has two phases. Authoring phase which happens in the decision central and ends with hitting Build & deploy (short path) or creating a container on decision server from a built project (longer path but this is also done through a UI screen in decision central).

When a KIE container is created from a project on the decision server, the second, deployment, phase starts. You can think of creating a KIE container as deploying or starting the project. KIE container represents a running project and you can create multiple KIE container from a single released (built) project.

The KIE container contains all assets from the project (rules, data objects, solver config etc.). Once you have a container created on the decision server, you can access its capabilities (backed by the assets it contains) through the decision server's remote APIs (typically REST but JMS is another available remote interface). Now you can create a solver from the solver configuration file (asset). Solver configuration is a template or matrix from which multiple (identical) solvers can be created. When a solver has been created you can submit a planning problem (aka solution instance) to it and ask for a current best solution at any time. Each solver can only optimize 1 planning problem at a time.

Creating containers, solvers and submitting problems through decision server REST API is already covered here: https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.0/html/installing_and_configuring_red_hat_business_optimizer/optimizer-rest-api-configuration-ref